debezium-core/src/main/java/io/debezium/transforms/TimezoneConverter.java

```diff
@@ -0,0 +1,470 @@
+        }
+        else if (!includeList.isEmpty()) {
+            collectTablesAndTopics(includeList);
+        }
+    }
+
+    private void collectTablesAndTopics(List<String> list) {
+        String commonPrefix = null;
+        for (String item : list) {
+            FieldItem parseItem = parseItem(item);
+            String prefix = parseItem.prefix;
+            String matchName = parseItem.getMatchName();
+            String field = parseItem.getFieldName();
+
+            if (prefix != null) {
+                commonPrefix = prefix;
+            }
+
+            if (Objects.equals(commonPrefix, TOPIC)) {
+                if (!topicFieldsMap.containsKey(matchName)) {
+                    topicFieldsMap.put(matchName, new HashSet<>());
+                }
+                topicFieldsMap.get(matchName).add(field);
+            }
+            else if (Objects.equals(commonPrefix, SOURCE)) {
+                if (!tableFieldsMap.containsKey(matchName)) {
+                    tableFieldsMap.put(matchName, new HashSet<>());
+                }
+                tableFieldsMap.get(matchName).add(field);
+            }
+            else {
+                if (!noPrefixFieldsMap.containsKey(matchName)) {
+                    noPrefixFieldsMap.put(matchName, new HashSet<>());
+                }
+                noPrefixFieldsMap.get(matchName).add(field);
+            }
+        }
+    }
+
+    private void validateConfiguration() {
+        if (!includeList.isEmpty()) {
+            if (!LIST_PATTERN.matcher(includeList.toString()).matches()) {
+                throw new DebeziumException(
+                        "Invalid include list format. Please specify a list of rules in the format of \"source:<tablename>:<fieldnames>\", \"topic:<topicname>:<fieldnames>\", \"<matchname>:<fieldnames>\"");
+            }
+        }
+        else if (!excludeList.isEmpty()) {
+            if (!LIST_PATTERN.matcher(excludeList.toString()).matches()) {
+                throw new DebeziumException(
+                        "Invalid exclude list format. Please specify a list of rules in the format of \"source:<tablename>:<fieldnames>\", \"topic:<topicname>:<fieldnames>\", \"<matchname>:<fieldnames>\"");
+            }
+        }
+
+        if (!TIMEZONE_PATTERN.matcher(convertedTimezone).matches()) {
+            throw new DebeziumException("Invalid timezone format. Please specify a timezone in the format of +/-HH:MM.");
+        }
+
+        if (!includeList.isEmpty() && !excludeList.isEmpty()) {
+            throw new DebeziumException("Both include and exclude lists are specified. Please specify only one.");
+        }
+    }
+
+    @Override
+    public void close() {
+    }
+
+    private enum Type {
+        ALL,
+        INCLUDE,
+        EXCLUDE
+    }
+
+    private Object getTimestampWithTimezone(String schemaName, Object fieldValue) {
+        Object updatedFieldValue = fieldValue;
+        ZoneId zoneId = ZoneId.of(convertedTimezone);
```

Comment:
How this behave with the DST? Is it correctly managed?

Reply:
For example in Italy we usually have the UTC+1 offeset but from the 25 March to 30 October we move to DST that is UTC+2. I am wondering if using a fixed offeset is right. https://www.baeldung.com/java-zone-offset

Reply:
I think yes since without think about DST can lead to wrong date conversion. 

You are using ZoneId.of("+01:00") and as for Java [docs](https://docs.oracle.com/javase/8/docs/api/java/time/ZoneId.html#of-java.lang.String-) in this case a [ZoneOffset.of(String)](https://docs.oracle.com/javase/8/docs/api/java/time/ZoneOffset.html#of-java.lang.String-) is used to parse the Zone ID. 

From ZoneOffset docs
>Different parts of the world have different time-zone offsets. The rules for how offsets vary by place and time of year are captured in the [ZoneId](https://docs.oracle.com/javase/8/docs/api/java/time/ZoneId.html) class.
For example, Paris is one hour ahead of Greenwich/UTC in winter and two hours ahead in summer. The ZoneId 
instance for Paris will reference two ZoneOffset instances - a +01:00 instance for winter, and a +02:00 instance for 
summer.

I think thath you need to use the same approach used in line #238 
```java
zoneId.getRules().getOffset(Instant.now())
```
also for the other case in the switch. 

I suggest to create specific test to see if it works.

Reply:
@mfvitale Thanks for raising the concern but I'd prefer to keep it off this PR and make a subseqent one.
We need to evaluate first if there is any impact on DST on values as theu come from Debezium core. If it is already accounted fro then it would be unnecessary to handle it here.

Reply:
How we store date in Debezium events? In case we are storing in Debezium zoned date consider the case when you want to move from a timezone that doesn't have the DST to another one that has it. In this case this cannot work.

Reply:
Thanks @mfvitale and @Naros for you inputs and concerns regarding DST. As discussed with @jpechane, I would be adding the support for both timezone names and timeshift values `+/-HH:MM`. It would be best suited for users and avoid unnecessary confusion.

Reply:
@mfvitale  @Naros How do we handle the tests when the DST is in effect?